File: E-EstateV2/src/app/_interface/fieldProduction.ts
export interface FieldProduction {
    id: number
    monthYear: string | null
    cuplump: number | null
    cuplumpDRC: number | null
    latex: number | null
    latexDRC: number | null
    uss: number | null
    ussDRC: number | null
    others: number | null
    othersDRC: number | null
    noTaskTap: number | null
    noTaskUntap: number | null
    fieldName: string
    fieldId: number
    createdBy: string
    createdDate: Date
    updatedBy: string
    updatedDate: Date
    remarkUntap: string
    totalTask: number
    estateId: number
    totalProduction: number
    totalCuplump: number
    totalLatex: number
    cuplumpDry: number
    latexDry: number
    ussDry: number
    othersDry: number
    tappedAreaPerHa: number
    isAcive: boolean
    status:string
    totalDry:number
}

File: E-EstateV2 API 2/E-EstateV2 API/Models/FieldProduction.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace E_EstateV2_API.Models
{
    public class FieldProduction
    {
        [Key]
        public int Id { get; set; }
        public string monthYear { get; set; }
        public float cuplump { get; set; }
        public float cuplumpDRC { get; set; }
        public float latex { get; set; }
        public float latexDRC { get; set; }
        public int noTaskTap { get; set; }
        public int noTaskUntap { get; set; }
        public string remarkUntap { get; set; }
        public string createdBy { get; set; }
        public DateTime createdDate { get; set; }
        public string updatedBy { get; set; }
        public DateTime updatedDate { get; set; }
        public string status { get; set; }


        [ForeignKey("FieldId")]
        public int fieldId { get; set; }
        public Field Field { get; set; }
    }
}
